index.js

```diff
@@ -49,6 +50,8 @@ var CONFIG = {
     }
 };
 
+var configFilePath = __dirname + "/.git-issues-config.json";
```

Comment:
Could you please store this file in the home directory? You can use the [`ul`](https://github.com/IonicaBizau/node-ul) package and do:

```js
var ul = require("ul");
...
var configFilePath = ul.USER_DIR + "/.git-issues-config.json";
```